src/saveload/saveload.h

```diff
@@ -345,6 +345,8 @@ enum SaveLoadVersion : uint16 {
 	SLV_MULTITRACK_LEVEL_CROSSINGS,         ///< 302  PR#9931 v13.0  Multi-track level crossings.
 	SLV_NEWGRF_ROAD_STOPS,                  ///< 303  PR#10144 NewGRF road stops.
 	SLV_LINKGRAPH_EDGES,                    ///< 304  PR#10314 Explicitly store link graph edges destination, PR#10471 int64 instead of uint64 league rating
+	SLV_SERVICE_INTERVAL_PERCENT_ONLY,      ///< 305  PR#10575 Don't use days for service intervals, only percents.
```

Comment:
I just noticed while reviewing this that savegame versions after 205 aren't grouped by 5's; there should be a second newline after line 342 / SLV_299 and another after line 347 / SLV_304.

Reply:
Thanks, I've opened a separate PR to fix that.